kube/.drone.yml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Will this still block the other build if there's a failure on this step?

Reply:
I think the answer is no, but I think that's fine for this project...